server/src/main/java/org/elasticsearch/persistent/PersistentTaskParams.java

```diff
@@ -16,5 +16,13 @@
  * Parameters used to start persistent task
  */
 public interface PersistentTaskParams extends VersionedNamedWriteable, ToXContentObject {
-
+    /**
+     * Lightweight tasks are allowed to be scheduled to nodes shutting down iff all nodes are shutting down.
+     * The setting
+     * {@link PersistentTasksClusterService#CLUSTER_TASKS_ALLOCATION_ALLOW_LIGHTWEIGHT_ASSIGNMENTS_TO_NODES_SHUTTING_DOWN_SETTING}
+     * needs to be set for this behavior to apply.
+     */
+    default boolean isLightweight() {
```

Comment:
I am guessing this term was inspired by the original issue. However, this term is not established and it was more used to describe the health node task. Since we do not have any other usage for this method, I suggest we give a more specific name, something along the lines of `keepRunningDuringShutdown` or `canRunOnShuttingDownNodes`. What do you think?

Reply:
++ for `canRunOnShuttingDownNodes`